### PR TITLE
distcc: add new package at 3.3.3

### PR DESCRIPTION
**Pull request by cemkeylan**
Please fill out the template.

Read: https://k1ss.org/style
Read: https://k1ss.org/guidestones

---

## Description of package
`distcc` is a program for sharing C/C++/ObjC compilation with other systems on the network.

I did not add Python as a dependency as it is not required unless you are a build server.

## Installed manifest of package

(`cat /var/db/kiss/installed/<pkg_name>/manifest`)

```
/var/db/kiss/installed/distcc/version
/var/db/kiss/installed/distcc/sources
/var/db/kiss/installed/distcc/manifest
/var/db/kiss/installed/distcc/etcsums
/var/db/kiss/installed/distcc/depends
/var/db/kiss/installed/distcc/checksums
/var/db/kiss/installed/distcc/build
/var/db/kiss/installed/distcc/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/pump.1.gz
/usr/share/man/man1/lsdistcc.1.gz
/usr/share/man/man1/include_server.1.gz
/usr/share/man/man1/distccmon-text.1.gz
/usr/share/man/man1/distccd.1.gz
/usr/share/man/man1/distcc.1.gz
/usr/share/man/man1/
/usr/share/man/
/usr/share/
/usr/lib/python3.8/site-packages/include_server/statistics.py
/usr/lib/python3.8/site-packages/include_server/setup.py
/usr/lib/python3.8/site-packages/include_server/run.py
/usr/lib/python3.8/site-packages/include_server/parse_file_test.py
/usr/lib/python3.8/site-packages/include_server/parse_file.py
/usr/lib/python3.8/site-packages/include_server/parse_command_test.py
/usr/lib/python3.8/site-packages/include_server/parse_command.py
/usr/lib/python3.8/site-packages/include_server/mirror_path_test.py
/usr/lib/python3.8/site-packages/include_server/mirror_path.py
/usr/lib/python3.8/site-packages/include_server/macro_eval_test.py
/usr/lib/python3.8/site-packages/include_server/macro_eval.py
/usr/lib/python3.8/site-packages/include_server/include_server_test.py
/usr/lib/python3.8/site-packages/include_server/include_server.py
/usr/lib/python3.8/site-packages/include_server/include_analyzer_test.py
/usr/lib/python3.8/site-packages/include_server/include_analyzer_memoizing_node_test.py
/usr/lib/python3.8/site-packages/include_server/include_analyzer_memoizing_node.py
/usr/lib/python3.8/site-packages/include_server/include_analyzer.py
/usr/lib/python3.8/site-packages/include_server/distcc_pump_c_extensions.cpython-38-x86_64-linux-gnu.so
/usr/lib/python3.8/site-packages/include_server/compress_files.py
/usr/lib/python3.8/site-packages/include_server/compiler_defaults.py
/usr/lib/python3.8/site-packages/include_server/cache_basics.py
/usr/lib/python3.8/site-packages/include_server/c_extensions_test.py
/usr/lib/python3.8/site-packages/include_server/basics_test.py
/usr/lib/python3.8/site-packages/include_server/basics.py
/usr/lib/python3.8/site-packages/include_server/__pycache__/statistics.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/setup.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/run.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/parse_file_test.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/parse_file.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/parse_command_test.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/parse_command.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/mirror_path_test.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/mirror_path.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/macro_eval_test.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/macro_eval.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/include_server_test.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/include_server.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/include_analyzer_test.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/include_analyzer_memoizing_node_test.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/include_analyzer_memoizing_node.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/include_analyzer.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/compress_files.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/compiler_defaults.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/cache_basics.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/c_extensions_test.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/basics_test.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/basics.cpython-38.pyc
/usr/lib/python3.8/site-packages/include_server/__pycache__/
/usr/lib/python3.8/site-packages/include_server/
/usr/lib/python3.8/site-packages/include_server-3.3.3-py3.8.egg-info
/usr/lib/python3.8/site-packages/
/usr/lib/python3.8/
/usr/lib/
/usr/bin/update-distcc-symlinks
/usr/bin/pump
/usr/bin/lsdistcc
/usr/bin/distccmon-text
/usr/bin/distccd
/usr/bin/distcc
/usr/bin/
/usr/
/etc/distcc/hosts
/etc/distcc/commands.allow.sh
/etc/distcc/clients.allow
/etc/distcc/
/etc/default/distcc
/etc/default/
/etc/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/distcc/build [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -e
+
+# Running distcc as a server instead of a client
+# requires python to be installed
```
Comment (dylanaraps) [inline community/distcc/build:4]: Full stop `.`

Comment (cemkeylan) [reply community/distcc/build:4]: @dylanaraps Should I add and resend a PR?

Comment (dylanaraps) [reply community/distcc/build:4]: Oops. I merged it anyway didn't I? That's my bad, apologies. If you like, we can fix this when the next release of `distcc` comes out.

Comment (dylanaraps) [reply community/distcc/build:4]: Up to you!

Comment (cemkeylan) [reply community/distcc/build:4]: distcc doesn't make releases often, so I'll send a PR now to not forget about it later :P

Comment (dylanaraps) [reply community/distcc/build:4]: Sounds good. :+1:

**Pull request merged**